### PR TITLE
[feature](cloud) Retry when FE receives E-230 msg in cloud mode

### PR DESCRIPTION
**Pull request by deardeng**
## Proposed changes

Issue Number: close #xxx

## Further comments

If this is a relatively large or complex change, kick off the discussion at [dev@doris.apache.org](mailto:dev@doris.apache.org) by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

**Comment by deardeng** [comment]
run buildall

File: fe/fe-core/src/main/java/org/apache/doris/qe/StmtExecutor.java [hunk 1]
```diff
@@ -781,6 +782,9 @@ private void handleQueryWithRetry(TUniqueId queryId) throws Exception {
                 handleQueryStmt();
                 break;
             } catch (RpcException | UserException e) {
+                if (Config.isCloudMode() && e.getMessage().contains("E-230")) {
```
Comment (morrySnow) [inline fe/fe-core/src/main/java/org/apache/doris/qe/StmtExecutor.java:785]: hard code string is not a good idea for maintainance

Comment (deardeng) [reply fe/fe-core/src/main/java/org/apache/doris/qe/StmtExecutor.java]: fixed

**Comment by deardeng** [comment]
run buildall

**Review by gavinchou — APPROVED** [review]

**Review by dataroaring — APPROVED** [review]
LGTM

**Pull request merged**